packages/core/src/common/server/web3-rpc/get-token-meta.ts

```diff
@@ -6,6 +6,19 @@ import { erc20Abi } from 'viem';
 import { web3Client } from './client';
 import { db } from '@hypha-platform/storage-postgres';
 
+function getIconForHyphaTokens(symbol: string, fallback: string): string {
```

Comment:
It would be nice to add a default fallback
```suggestion
function getIconForHyphaTokens(symbol: string, fallback: string = '/placeholder/default-token.svg'): string {
```